source/libraries/ContractDeployer.ts

```diff
@@ -134,7 +135,7 @@ export class ContractDeployer {
 
         const promises: Promise<ContractBlockchainData|undefined>[] = [];
         for (let contractFileName in this.compiledContracts) {
-            if (contractFileName === "Controller.sol" || contractFileName === "libraries/Delegator.sol") {
+            if (contractFileName === "Controller.sol" || contractFileName === "libraries/Delegator.sol" || contractFileName.startsWith('legacy_reputation/')) {
```

Comment:
We should never have been uploading the old legacy rep contracts.  We should also remove them from compilation, but that required more understanding then I wanted to figure out at the moment.